jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -570,6 +576,7 @@ void java_bytecode_parsert::get_class_refs()
 
   for(const auto &method : parse_tree.parsed_class.methods)
   {
+    get_annotation_refs(method.annotations);
```

Comment:
You may also like to iterate over the method parameters and grab the annotations from those too.